### ISSUE TITLE
Combine thumbstands into two pieces

### ISSUE THREAD

**Issue by 7t2** [nickcoutsos/dactyl-flatpacked#4]
![thumbstands](https://user-images.githubusercontent.com/103980579/164130014-b3a86d8a-e649-4de8-9bf3-d142de47f374.png)

Having the thumbstands as 6 individual pieces makes assembly really difficult. If they could be combined into just two pieces, it would make assembly alot easier and it would save on print space. It's a win win

**Comment by nickcoutsos** [comment]
Yeah, this one is pretty unfortunate but I tried for a long time to avoid it. My first dactyl had just the front and back cross supports for the thumb cluster and I ended up cracking a number of pieces (I ended up having to replace them with 3D printed parts). It was modeled in 3D with slot cutouts made from boolean subtractions, and I didn't notice that the cuts weren't actually orthogonal. I later "corrected" it by switching the order of some matrix transformations, but that wasn't right either.

Apologies in advance because I don't think I understand the math well enough to explain it correctly, but I'm going to try.

1. All pieces that slot together need to intersect at 90˚ so that slot can be cut by a laser
2. Cross supports must intersect the XY plane at 90˚ so that they'll sit flat on table.

This seems to be easy for the finger cluster, but the thing that screws up the second requirement is the fact that the Dactyl thumb cluster has a slight rotation on the axis `[1, 1, 0]` (relevant line in [dactyl-flatpacked](https://github.com/nickcoutsos/dactyl-flatpacked/blob/56549649684b9b4ed242cfab8232d149fab0e2cd/positioning.scad#L58) and the original [dactyl-keyboard](https://github.com/adereth/dactyl-keyboard/blob/e0416e329032b3648a86e53656906702603dbeb0/src/dactyl_keyboard/dactyl.clj#L261).

I think this is another case where a base piece is needed to hold all the cross supports in place, but other than that I can't think of a way to reduce the number of cross supports here without sacrificing the thumb cluster tilt.

**Comment by 7t2** [comment]
Mathematics is my major so I understand what you mean. If each tower has slightly more rotation than the last around the (1, 1, 0) axis, then leave the 6 parts individual and give them each a slot into the base to sit into, then you ideally wouldn't need to attach them together with supports.

I guess the joint would be the acrylic analogue of the mortice and tenon joint, I'm not sure how to describe it so I put a little image together. I exaggerated the rotation just for visualisation, but I think we both know what I mean

![thumbToBase](https://user-images.githubusercontent.com/103980579/164214485-679fcf81-4f77-4c24-8f4b-7bea1a362f52.png)

**Comment by nickcoutsos** [comment]
So if I'm understanding the size of the cutouts correctly, the front view of one of those vertical pieces would look like

```
 _  ___  _
//_/  / //
|        |
|        |
|        |
|        |
|        |
|_      _|
  |____|    <-- this is the joint
```

_Very_ rough diagram.

I hadn't given it a lot of thought but I think this will work best due to how close those vertical support are packed.

**Comment by 7t2** [comment]
That's exactly what I mean, that should provide enough stability for the thumb cluster, but if not, we can cross that bridge when we come to it
